releaseNotes/JavaScriptFeatureManagementApplicationInsightsBrowser.md

```diff
@@ -0,0 +1,13 @@
+# Microsoft Feature Management for JavaScript
```

Comment:
How do you like this?

Microsoft JavaScript Feature Management for Application Insights in Browsers